subsys/spm/secure_services.c

```diff
@@ -66,7 +68,7 @@ struct read_range {
 
 
 __TZ_NONSECURE_ENTRY_FUNC
-int spm_request_read(void *destination, uint32_t addr, size_t len)
+int spm_request_read_nsc(void *destination, uint32_t addr, size_t len)
```

Comment:
I've recommended renaming this from "nsc" to "nse" for **N**on-**S**ecure **E**ntry to conform to the macro used

Reply:
Thanks, done